client/landing/stepper/declarative-flow/internals/steps-repository/sensei-plan/styles.scss

```diff
@@ -0,0 +1,127 @@
+@import "@automattic/onboarding/styles/mixins";
+
+$sensei-color: #43af99;
+
+.sensei .progress-bar__progress {
+	background-color: $sensei-color;
+}
```

Comment:
Similarly I see some of the component overrides multiple times (like this `.progress-bar__progress` override). There should be a more centralized way to do the same I'm sure.

Reply:
Moved to a common file here: 2b105cd55e551c7b35b817c6e06caa967c148645